tests/integration/test_basic.py

```diff
@@ -177,6 +151,30 @@ def test_store_pii_stripping(mini_sentry, relay):
     assert event["message"] == "[email]"
 
 
+def test_event_timeout(mini_sentry, relay):
```

Comment:
Actually this test is flaky for me. Try this:

```git checkout origin/master server/
py.test tests/integration --reruns 5 -kevent_timeout
```

this should fail five times, right? But it seems to pass everytime.

`sleep(2)` instead of `sleep(1.5)` fixes this for me.

Reply:
This is either a bug or some really strange delays on your end. What does the test fail with?

Reply:
Sorry, I forgot one command. Basically I want to verify that the test does fail if I revert your changes to the server